test/dummy-jira.js

```diff
@@ -58,10 +59,30 @@ class DummyJira {
       SubTask13: issueGenerator('SubTask13', 'Sub-task', 'green', 'MT1'),
       SubTask14: issueGenerator('SubTask14', 'Sub-task', 'green', 'Bug1'),
       SubTask15: issueGenerator('SubTask15', 'Sub-task', 'green'),
+
+      //test to see if the code cares about broken children
+      //SubTask16: issueGenerator('SubTask16', 'Sub-task', 'red','Story1'),
       //SubTask1: issueGenerator('SubTask1', 'Sub-task', 'yellow'), // Missing tests?
 
       Task1: issueGenerator('Task1', 'Task', 'yellow'),
 
+      //new code
```

Comment:
The diff generated by github also tells me this is new code. We don't need this comment.